apis/python/tests/test_sparse_nd_array.py

```diff
@@ -84,33 +85,38 @@ def test_sparse_nd_array_reopen(tmp_path):
         tmp_path.as_posix(), type=pa.float64(), shape=(1,), tiledb_timestamp=1
     )
 
-    with soma.SparseNDArray.open(tmp_path.as_posix(), "r") as A1:
+    with soma.SparseNDArray.open(tmp_path.as_posix(), "r", tiledb_timestamp=1) as A1:
         with raises_no_typeguard(ValueError):
             A1.reopen("invalid")
 
-        with A1.reopen("w") as A2:
-            with A2.reopen("r") as A3:
+        with A1.reopen("w", tiledb_timestamp=2) as A2:
+            with A2.reopen("r", tiledb_timestamp=3) as A3:
                 assert A1.mode == "r"
                 assert A2.mode == "w"
                 assert A3.mode == "r"
-                assert A1.tiledb_timestamp < A2.tiledb_timestamp
-                assert A2.tiledb_timestamp < A3.tiledb_timestamp
-
-    with soma.SparseNDArray.open(tmp_path.as_posix(), "r") as A1:
-        with A1.reopen("r") as A2:
+                assert A1.tiledb_timestamp.timestamp() == 0.001
```

Comment:
I'd prefer `A1.tiledb_timestamp_ms` so we can do int-to-int compares rather than relying on whether floating-point equality rounds correctly (there's no need for floating-point here)